tsl/test/isolation/expected/detach_chunk_isolation.out

```diff
@@ -0,0 +1,215 @@
+Parsed test spec with 4 sessions
+
+starting permutation: s2_wait_before_lock s1b s1_detach s2_show_num_rows s3_insert s2_show_num_rows s2_release_lock s3_insert s1c s2_show_num_rows s2_show_num_chunks s2_show_chunk_rows
+step s2_wait_before_lock: SELECT debug_waitpoint_enable('chunk_detach_before_lock');
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s1b: BEGIN;
+step s1_detach: CALL detach_chunk('chunk_to_detach'); <waiting ...>
+step s2_show_num_rows: SELECT count(*) FROM detach_test;
+count
+-----
+    1
+(1 row)
+
+step s3_insert: INSERT INTO detach_test VALUES ('2025-06-02 10:30', 1, 31.5);
+step s2_show_num_rows: SELECT count(*) FROM detach_test;
+count
+-----
+    2
+(1 row)
+
+step s2_release_lock: SELECT debug_waitpoint_release('chunk_detach_before_lock');
+debug_waitpoint_release
+-----------------------
+                       
+(1 row)
+
+step s1_detach: <... completed>
+step s3_insert: INSERT INTO detach_test VALUES ('2025-06-02 10:30', 1, 31.5); <waiting ...>
+step s1c: COMMIT;
+step s3_insert: <... completed>
+ERROR:  chunk not found
+step s2_show_num_rows: SELECT count(*) FROM detach_test;
+count
+-----
+    0
+(1 row)
+
+step s2_show_num_chunks: SELECT count(*) FROM show_chunks('detach_test');
+count
+-----
+    0
+(1 row)
+
+step s2_show_chunk_rows: SELECT count(*) FROM chunk_to_detach;
+count
+-----
+    2
+(1 row)
+
+
+starting permutation: s2_wait_before_lock s1b s1_detach s2_show_num_rows s2_show_num_chunks s3_truncate s2_release_lock s1c s2_show_num_chunks s2_show_chunk_rows
+step s2_wait_before_lock: SELECT debug_waitpoint_enable('chunk_detach_before_lock');
+debug_waitpoint_enable
+----------------------
+                      
+(1 row)
+
+step s1b: BEGIN;
+step s1_detach: CALL detach_chunk('chunk_to_detach'); <waiting ...>
+step s2_show_num_rows: SELECT count(*) FROM detach_test;
+count
+-----
+    1
+(1 row)
+
+step s2_show_num_chunks: SELECT count(*) FROM show_chunks('detach_test');
+count
+-----
+    1
+(1 row)
+
+step s3_truncate: TRUNCATE TABLE detach_test;
+step s2_release_lock: SELECT debug_waitpoint_release('chunk_detach_before_lock');
+debug_waitpoint_release
+-----------------------
+                       
+(1 row)
+
+step s1_detach: <... completed>
+ERROR:  chunk not found
```

Comment:
I guess we implemented TRUNCATE by dropping chunks, so that's why it is complaining about "chunk not found", correct?